plugins/woocommerce/src/Internal/Admin/ReviewsListTable.php

```diff
@@ -202,10 +202,38 @@ private function get_item_author_url_for_display( $author_url ) {
 	/**
 	 * Renders the "submitted on" column.
 	 *
-	 * @param object|array $item Review or reply being rendered.
+	 * Note that the output is consistent with {@see WP_Comments_List_Table::column_date()}.
+	 *
+	 * @param WP_Comment $item Review or reply being rendered.
 	 */
 	protected function column_date( $item ) {
-		// @TODO Implement in MWC-5338 {agibson 2022-04-12}
+
+		$submitted = sprintf(
+			/* translators: 1 - Product review date, 2: Product review time. */
+			__( '%1$s at %2$s', 'woocommerce' ),
+			/* translators: Review date format. See https://www.php.net/manual/datetime.format.php */
+			get_comment_date( __( 'Y/m/d', 'woocommerce' ), $item ),
+			/* translators: Review time format. See https://www.php.net/manual/datetime.format.php */
+			get_comment_date( __( 'g:i a', 'woocommerce' ), $item )
```

Comment:
Is it normal to make these translateable?

I wonder if it's a better idea to check with product to see if we can use the site's date + time format settings instead of hard-coding something. That's a better user experience IMO.

Reply:
I have copied these from `WP_Comments_List_Table::column_date()` and mentioned in the phpdoc it's a 1:1 implementation with the comments page. The product team mockup seems also hinting at the same output for this column WRT the date format. Product Reviews Pro has a different handling.

Reply:
I still think the site formats would be better, but I'll let it go since this does indeed just match the normal comments behaviour.

Reply:
I agree with you - but it's easy to change if product or A8C want to follow with `wc_date_format` and `wc_time_format`